digiroad2-api-common/src/main/scala/fi/liikennevirasto/digiroad2/Digiroad2Context.scala

```diff
@@ -412,6 +437,16 @@ object Digiroad2Context {
   val pedestrianCrossingVerifier = system.actorOf(Props(classOf[PedestrianCrossingValidation], pedestrianCrossingValidator), name = "pedestrianCrossingValidator")
   eventbus.subscribe(pedestrianCrossingVerifier, "pedestrianCrossing:Validator")
 
+  val lanesUpdater = system.actorOf(Props(classOf[LanesUpdater], laneService), name = "lanesUpdater")
+  eventbus.subscribe(lanesUpdater, "lanes:updater")
+
+  val lanesGenerator = system.actorOf(Props(classOf[LanesGenerator[PersistedLane]], laneService), name = "lanesGenerator")
+  eventbus.subscribe(lanesGenerator, "lanes:generator")
+
```

Comment:
This LaneGenarator is just a duplicate behaviour as LaneSaveProjected

Reply:
LaneGenerator deleted now aggregated with the old LaneSaveProjected